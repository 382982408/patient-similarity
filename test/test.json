[{"features": [{"id": "HP:0000365"}, {"id": "HP:0002474"}, {"id": "HP:0000358"}, {"id": "HP:0004322"}, {"id": "HP:0000448"}, {"id": "HP:0001510"}, {"id": "HP:0000527"}, {"id": "HP:0000213"}, {"id": "HP:0000325"}], "external_id": "FHS 1", "report_id": "P0000001"}, {"features": [{"id": "HP:0000689"}, {"id": "HP:0003799"}, {"id": "HP:0003510"}, {"id": "HP:0004325"}, {"id": "HP:0009381"}, {"id": "HP:0004322"}, {"id": "HP:0000822"}, {"id": "HP:0000219"}, {"id": "HP:0000646"}, {"id": "HP:0000483"}, {"id": "HP:0002808"}, {"id": "HP:0003149"}, {"id": "HP:0000581"}, {"id": "HP:0001764"}, {"id": "HP:0010230"}, {"id": "HP:0000368"}, {"id": "HP:0000470"}, {"id": "HP:0000691"}, {"id": "HP:0200055"}, {"id": "HP:0002194"}, {"id": "HP:0005616"}, {"id": "HP:0000750"}, {"id": "HP:0001620"}, {"id": "HP:0000154"}, {"id": "HP:0000325"}], "external_id": "FHS 2", "report_id": "P0000002"}, {"features": [{"id": "HP:0008070"}, {"id": "HP:0000252"}, {"id": "HP:0000275"}, {"id": "HP:0000506"}, {"id": "HP:0000527"}, {"id": "HP:0002213"}, {"id": "HP:0004322"}, {"id": "HP:0001508"}, {"id": "HP:0000316"}, {"id": "HP:0005274"}, {"id": "HP:0000960"}, {"id": "HP:0000348"}, {"id": "HP:0000431"}, {"id": "HP:0000369"}, {"id": "HP:0000340"}, {"id": "HP:0005338"}, {"id": "HP:0002205"}, {"id": "HP:0000750"}, {"id": "HP:0004325"}, {"id": "HP:0000670"}, {"id": "HP:0011968"}, {"id": "HP:0000612"}, {"id": "HP:0000958"}], "external_id": "Dubowitz 1", "report_id": "P0000003"}, {"features": [{"id": "HP:0000175"}, {"id": "HP:0000431"}, {"id": "HP:0000448"}, {"id": "HP:0000670"}, {"id": "HP:0000347"}, {"id": "HP:0002209"}, {"id": "HP:0000369"}, {"id": "HP:0000325"}, {"id": "HP:0002007"}, {"id": "HP:0001054"}, {"id": "HP:0001256"}, {"id": "HP:0000581"}, {"id": "HP:0004325"}, {"id": "HP:0001620"}, {"id": "HP:0000316"}, {"id": "HP:0000508"}, {"id": "HP:0002650"}, {"id": "HP:0000612"}], "external_id": "Dubowitz 2", "report_id": "P0000004"}]