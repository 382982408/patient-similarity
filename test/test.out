#A	B	simgic
FHS 1	FHS 2	0.142661
FHS 1	Dubowitz 1	0.194089
FHS 1	Dubowitz 2	0.169846
FHS 2	Dubowitz 1	0.122301
FHS 2	Dubowitz 2	0.163740
Dubowitz 1	Dubowitz 2	0.294508
